genienlp/arguments.py

```diff
@@ -192,10 +195,8 @@ def post_parse(args):
         for t in args.train_task_names:
             if t not in args.val_task_names:
                 args.val_task_names.append(t)
-    if 'imdb' in args.val_task_names:
-        args.val_task_names.remove('imdb')
 
-    args.timestamp = datetime.datetime.now(tz=datetime.timezone.utc).isoformat()
+    args.timestamp = datetime.datetime.now(tz=datetime.timezone.utc).strftime('%D-%H:%M:%S %Z')
```

Comment:
Why these two changes?
(imdb does not have validation data so the previous change was correct)

Reply:
I added the imdb back.
The `strftime` formatting reduces the width of log messages a bit.